web/main.ts

```diff
@@ -57,11 +58,51 @@ function createWindow(): BrowserWindow {
   return win;
 }
 
+const startBinary = () => {
+  const tmpPath = path.join(os.tmpdir(), 'octant');
+  fs.mkdir(path.join(tmpPath), { recursive: true }, (err) => {
+    if (err) {
+      throw err;
+    }
+  });
+
+  const out = fs.openSync(path.join(tmpPath, 'api.out.log'), 'a');
+  const err = fs.openSync(path.join(tmpPath, 'api.err.log'), 'a');
+
+  let serverBinary: string;
+  if (local) {
+    serverBinary = path.join(__dirname, 'extraResources', 'octant');
+  } else {
+    serverBinary = path.join(process.resourcesPath, 'extraResources', 'octant');
+  }
+
+  const server = child_process.spawn(serverBinary, [], {
+    env: { NODE_ENV: 'production', PATH: process.env.PATH },
+    detached: true,
+    stdio: ['ignore', out, err],
```

Comment:
```
const server = child_process.spawn(serverBinary, ['--disable-open-browser'], { // don't try to open the browser
    env: { ...process.env, NODE_ENV: 'production' }  // this makes sure all the environment variables are passed in
```

Reply:
Updated to pass flag and system env vars